ZKUI-396 // Veeam Configuration Summary Page

hervedombya opened the pull request.

hervedombya committed:
ZKUI-396 // Veeam Configuration Summary Page

Update Veeam UI elements and package dependencies

Update dependencies and fix minor code issues

Add Veeam constants and update Veeam configuration
UI

bert-e commented:
# Hello hervedombya,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko-ui/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZKUI-396 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.0`


Please check the `Fix Version/s` of ZKUI-396, or the target
branch of this pull request.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @jbwatenbergscality

* @monpote

ChengYanJin commented on src/react/ui-elements/Veeam/VeeamTable.tsx:
```diff
@@ -102,39 +101,38 @@ export default function VeeamTable() {
-            />
-          </Stack>
-        </Wrap>
+    <Form
```
We may want to place the table not within a form.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @jbwatenbergscality

* @chengyanjin

* @monpote

ChengYanJin committed:
ZKUI-396: Bump core-ui to 0.102.0

Attached the Stepper context to the window to make sure we reuse the same context even though the bundle load again

ChengYanJin committed:
ZKUI-396: Add the Stepper in the test

ChengYanJin committed:
ZKUI-396: Improve the layout of the Veeam Stepper

JBWatenbergScality commented:
/bypass_author_approval

bert-e commented:
# Not authorized



I'm afraid I cannot do that, @jbwatenbergscality:

> /bypass_author_approval


You don't have enough credentials to use `bypass_author_approval`.


Please **edit** or **delete** the corresponding comment so I can move on.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @monpote

* @chengyanjin

* @jbwatenbergscality

ChengYanJin reviewed (approved):
LGTM!!!

hervedombya commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.2`



The following branches will **NOT** be impacted:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`
* `development/2.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.2`



The following branches have **NOT** changed:

* `development/1.4`
* `development/1.5`
* `development/1.6`
* `development/2.0`
* `development/2.1`



Please check the status of the associated issue ZKUI-396.

Goodbye hervedombya.
